packages/bundle/src/index-es5.ts

```diff
@@ -13,8 +16,16 @@ import 'core-js/modules/es6.symbol';
 import 'core-js/modules/es7.array.includes';
 import 'url-search-params-polyfill';
 import 'whatwg-fetch';
-
 import addVersion from './addVersion';
+import defaultCreateDirectLine from './createDirectLine';
+import { version } from './index-minimal';
```

Comment:
Sorting.

Put `{ version }` above `addVersion`, because we are using line-sorter here, i.e. ASCII-ically, character `{` come before letter `a`.